packages/shared/src/utils/canvas.ts

```
@@ -235,6 +235,30 @@ export class UniCanvas {
+    }
+
+    if (typeof requestAnimationFrame === "function") {
+      return requestAnimationFrame(callback);
+    }
+
+    return setTimeout(callback, 1000 / 60);
+  }
+
+  cancelAnimationFrame(id: number): void {
+    if (this.canvasNode != null) {
+      return this.canvasNode.cancelAnimationFrame(id);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)